src/components/NavigationBar.tsx

```diff
@@ -1,155 +1,516 @@
+                        },
+                      }),
+                    }}
+                  >
+                    {subItem.label}
+                  </Typography>
+                ))}
+              </Box>
+            </Popover>
+          )}
+        </Box>
+      );
+    });
 
-  const paths = [
-    "/cittadini",
-    "/imprese",
-    "/pubbliche-amministrazioni",
-    "/punti-di-ritiro",
-    "/numeri",
-    "/faq",
+  const sendMenuItems = [
+    {
+      icon: <PeopleIcon sx={styles.sendMenuIcon} />,
+      title: "Cittadini",
+      description:
+        "Accedi come persona fisica, libero professionista o ditta individuale",
+      link: "https://cittadini.notifichedigitali.it/auth/login",
+    },
+    {
+      icon: <BusinessIcon sx={styles.sendMenuIcon} />,
+      title: "Imprese",
+      description: "Accedi come persona giuridica",
+      link: "https://imprese.notifichedigitali.it/auth/login",
+    },
   ];
 
-  useEffect(() => {
-    setIsMobileMenuOpen(false);
-    setIsSendMenuOpen(false);
-    setSubMenuAnchorEl(null);
-    setRitiroSubMenuAnchorEl(null);
+  const thirdElementContent = (
+    <Box
+      sx={{
+        border: "1px solid #E3E7EB",
+        borderRadius: 4,
+        padding: 2,
+        textAlign: "left",
+        bgcolor: "white",
+      }}
+    >
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        sx={{
+          marginBottom: 1,
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "space-between",
+        }}
+      >
+        <Typography variant="subtitle1" sx={{ fontWeight: 600 }}>
+          Sei un ente?
+        </Typography>
+        <Button
+          variant="text"
+          color="primary"
+          sx={styles.sendMenuButton}
+          onClick={() =>
+            (window.location.href = "https://selfcare.pagopa.it/auth/login")
+          }
+          endIcon={<ArrowForwardIcon />}
+        >
+          Accedi
+        </Button>
+      </Stack>
+      <Box sx={{ borderTop: "1px solid #E3E7EB", paddingTop: 1 }}>
+        <Typography variant="body2">
+          Il tuo ente non ha ancora aderito?{" "}
+          <a
+            href="https://selfcare.pagopa.it/auth/login?onSuccess=%2Fonboarding%2Fprod-pn"
+            style={{ color: "#0066CC", textDecoration: "underline" }}
+          >
+            Scopri come aderire
+          </a>
+        </Typography>
+      </Box>
+    </Box>
+  );
 
-    function handleResize() {
-      const mobile = window.innerWidth < 900;
-      setIsMobile(mobile);
-      if (!mobile) {
-        setIsMobileMenuOpen(false);
-        setIsSendMenuOpen(false);
-      }
-    }
-    handleResize();
-    window.addEventListener("resize", handleResize);
-    setMounted(true);
-    return () => {
-      window.removeEventListener("resize", handleResize);
-    };
-  }, [pathname]);
+  const renderSendMenu = () => (
+    <Box sx={styles.sendMenuBox}>
```

Comment:
it seems that only property bgcolor is needed from styles.sendMenuBox. Do the check and if it is correct, remove variable styles.sendMenuBox and copy bgcolor property here